### ISSUE TITLE
Refinements to a subset of lesion masks

### ISSUE THREAD

**Issue by pritesh-mehta** [rcuocolo/PROSTATEx_masks#7]
Hi Renato,

I have counted the number of components in each of the lesion mask nifti files. Most have one component as intended. However some have additional errant elements, so would request they be refined:

ProstateX0020-Finding1-t2_tse_tra_ROI.nii.gz | 2
ProstateX0025-Finding1-t2_tse_tra_ROI.nii.gz | 4
ProstateX0029-Finding1-t2_tse_tra_ROI.nii.gz | 3
ProstateX0032-Finding1-t2_tse_tra_ROI.nii.gz | 3
ProstateX0033-Finding3-t2_tse_tra_ROI.nii.gz | 2
ProstateX0050-Finding1-t2_tse_tra_ROI.nii.gz | 2
ProstateX0063-Finding1-t2_tse_tra_ROI.nii.gz | 2
ProstateX0067-Finding3-t2_tse_tra_ROI.nii.gz | 2
ProstateX0085-Finding3-t2_tse_tra_ROI.nii.gz | 2
ProstateX0086-Finding2-t2_tse_tra_ROI.nii.gz | 2
ProstateX0095-Finding1-t2_tse_tra_ROI.nii.gz | 2
ProstateX0099-Finding1-t2_tse_tra_ROI.nii.gz | 5
ProstateX0099-Finding2-t2_tse_tra_ROI.nii.gz | 2
ProstateX0104-Finding1-t2_tse_tra_ROI.nii.gz | 2
ProstateX0106-Finding2-t2_tse_tra_ROI.nii.gz | 2
ProstateX0165-Finding1-t2_tse_tra0_ROI.nii.gz | 3

Another issue is that the two clinically significant ProstateX0199 lesion masks drawn overlap when combined into a single volume. Can the ProstateX0199 lesion masks be adjusted so that they do not overlap.

As always many thanks for your efforts,
Pritesh

**Comment by rcuocolo** [comment]
Thank you for letting me know. I will check these masks again in the next days, I will upload corrected files as soon as possible. For patient 199, I will look again at the masks and original coordinates (corresponding 3x3 masks have now been included in the repository if you are interested) to assess how best to proceed.

**Comment by pritesh-mehta** [comment]
Re patient 199, the two masks seem to be in right place based on the coordinates. They are just overlapping slightly since they are both large and in the central gland. Thanks Renato.

**Comment by rcuocolo** [comment]
I checked the masks with multiple clusters, which were probably due to misclicks during the manual segmentation process. They have been cleaned and uploaded. The T2 lesion masks should only contain a single, correct cluster now.
I have also checked patient 199, I will correct the segmentation shortly in consensus with the other senior reader and upload the updated masks. I will let you know as soon as it is done.

**Comment by rcuocolo** [comment]
Sorry for the wait. Today I was finally able to work together with the other radiologist involved in the repository and we have redone the annotations for the T2 images of case 199. I believe they have been improved overall and the overlap should also be fixed. I would appreciate it if you could also confirm the masks are now correct.

**Comment by pritesh-mehta** [comment]
No problem. Thanks Renato. The masks for case 199 look good (to my untrained eye). The overlap has also gone as desired. However, there is still a point of contact between the masks on one of the slices (slice 9 I believe). I think the point of contact should be removed, else the masks are essentially describing one lesion. What are your thoughts?

**Comment by rcuocolo** [comment]
I have checked again, on slice 9 the masks become contiguous but do not overlap. The issue is that this lesion could actually also be considered one large lesion rather than two separate ones. In the T2w images it is very challenging to actually separate them, while it is clearer on ADC maps. However, in the TZ the T2w are the reference standard, while DWI/ADC has an ancillary role. Therefore, lesion extension is better reflected by T2w images. As the PROSTATEx dataset considers these 2 separate lesions and the biopsy cores are reported quite fairly apart according to lesion coordinates, we have kept this classification. We believe the current mask to be a good compromise to actually include all suspicious tissue according to PI-RADS criteria while also adhering to PROSTATEx labels. I would not edit it further, honestly.
